plaidml/bridge/openvino/ops/squeeze.cpp

```diff
@@ -19,7 +19,7 @@ void registerSqueeze() {
   registerOp("squeeze", [](const Context& ctx) {
     IE_ASSERT(ctx.operands.size() == 2);
     auto I = ctx.operands.at(0);
-    auto axes = get_axis_set_from_constant_operand(1, ctx.layer);
+    auto axes = get_constant_vector<int>(1, ctx.layer);
     std::vector<int> v_axes;
     v_axes.assign(axes.begin(), axes.end());
```

Comment:
This copy to v_axes seems redundant now.

Reply:
fixed